iomanage.py

```diff
@@ -0,0 +1,42 @@
+import pickle
+import os
+
+class IOManage:
+    """This class help you save matrix image in a file text"""
+
+    _path = None
+
+    def __init__(self, path):
+        self._path = path
```

Comment:
I think maybe it can be great to gave all these functions as `@staticmethod` so we can easily work with multiple file. Don't you think ?